modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/PublicationTimeline.js

```diff
@@ -16,6 +16,7 @@ const PublicationTimeline = ({timelineItems}) => {
 			<div className="publication-timeline">
 				{timelineItems.map((timelineItem) => (
 					<ClayPanel
```

Comment:
It might be simpler if we try to render the timeline items with [ClayList](https://clayui.com/docs/components/list.html). It allows you to define actions for each item in the list. Here is an example: https://storybook.clayui.com/?path=/story/design-system-components-list--list-with-items

```
const dropdownActions = [
	{
		label: 'clickable',
		onClick: () => {
			alert('you clicked!');
		},
	}
	{
		href: '#',
		label: 'linkable',
	},
];
```